src/webhook-handler/handler.js

```diff
@@ -36,12 +36,21 @@ module.exports.handleWebhookEvent = (event, context, callback) => {
 
 const handleSnsPublish = (event) => {
   const body = extractMessageBody(event)
-  const eventTopicArn = eventTopicData.get(body.event.value).sns_arn
+  // const eventTopicArn = eventTopicData.get(body.event.value).sns_arn
```

Comment:
Don't need to comment this out - just remove the line, VCS handles the rest.

Reply:
My bad, just forgot to remove it

Reply:
This is fixed in https://github.com/lulibrary/alma-webhook-handler/pull/16/commits/5b8ee7fbf1a39304611944de29983bf839ce1634